### DISCUSSION TITLE
Creating PR from issue on GH leaves PR without title in some places

### DISCUSSION THREAD

**Discussion by hexmode** [magit/forge#617] [Support] [answered]
There was an issue with a title and I used forge's "create pr from issue" to create a PR that would close it.

The url is now `.../pull/XXX` instead of `.../issue/XXX` as expected, and the title displays correctly on the page, but when I look at the PR in any other view (this is a private repo and we're using projects where the PRs are displayed in tables, etc.) an empty entry is shown.  For example:
![image](https://github.com/magit/forge/assets/43581/938ff096-e6f7-4a03-89a8-d4d231444ecf)
![image](https://github.com/magit/forge/assets/43581/a597aec4-f9be-43e0-831c-7b8e3bd8a953)

The title of the PR should be the title of the issue.

**Comment by tarsius** [comment]
Based on it being displayed "on the page", we know that Github is aware of the title. It seems to me that that means that Forge was successful in telling the API to convert the topic.

I don't know what feature is being shown in the cropped screenshots. I am guessing it is the Github's "project" feature, which I don't use and thus would not recognize. But it might also be a third-party tool.

It seems to me that this isn't a Forge issue. Github can display the title in some places and in others it cannot. To me that indicates that the issue is on their end.

**Reply by hexmode** [reply]
Screenshots are from [GH project](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) views.

You may be right that this is a GH issue, not a forge issue.  I thought it could be a forge issue because the title is not displayed on any of these project views -- maybe there is a way to provide the title to the PR when creating a PR from an issue?

**Reply by hexmode** [reply]
I created a project to demonstrate this. https://github.com/orgs/MWStake/projects/1/views/1

**Reply by hexmode** [reply]
I didn't manage to get a snapshot of this before I created a PR from the issue, but this box showed the title before I created the PR from the issue:
![image](https://github.com/magit/forge/assets/43581/419a19f1-593f-4ff1-94fe-5920c3b12a62)
When I created the PR and then switched back to the tab for the issue, I saw the project board dynamically update to remove the title.

**Reply by hexmode** [reply]
PR: https://github.com/MWStake/Website/pull/1

**Reply by hexmode** [reply]
Closing the PR without merging shows:
![image](https://github.com/magit/forge/assets/43581/cda819f1-e292-411c-9948-4d7a4ac87ab0)

**Reply by hexmode** [reply]
For fun, reopening the PR keeps the title:
![image](https://github.com/magit/forge/assets/43581/e86e5a59-be2e-4a14-adde-e04259c88fc7)

**Reply by hexmode** [reply, marked as answer]
so, at this point, unless I'm missing something, this looks like a GH issue.
